### PR TITLE
add swarm manage nodes guide

### PR DESCRIPTION
**Pull request by sfsmithcha**
add manage nodes to manage a swarm guide

Signed-off-by: Charles Smith charles.smith@docker.com

### CODE REVIEW

**Comment by sfsmithcha** [comment]
@aaronlehmann , @abronan PTAL

File: docs/swarm/manage-nodes.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+node.
+
+The `MANAGER STATUS` column shows node participation in the Raft consensus:
+
+* No value indicates a worker node that does not participate in swarm
+management.
+* `Leader` means the node is the primary manager node that makes all swarm
+management and orchestration decisions for the swarm.
+* `Reachable` means the node is a manager node is participating in the Raft
+concensus. If the leader node becomes unavailable, the node is eligible for
```
Comment (aaronlehmann) [inline docs/swarm/manage-nodes.md]: consensus is misspelled.

File: docs/swarm/manage-nodes.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+election as the new leader.
+* `Unavailable` means the node is a manager that is not able to communicate with
+other managers in the Raft. If a manager node becomes unavailable, you should
```
Comment (aaronlehmann) [inline docs/swarm/manage-nodes.md]: remove "in the Raft".

File: docs/swarm/manage-nodes.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+`docker node promote` and `docker node demote` are convenience commands for
+`docker node update --role manager` and `docker node update --role worker`
+respectively.
+
+
+## Remove a node from the swarm
+
+Before you can remove a node from the swarm, you must first run the
+`docker swarm leave` command on the none you want to remove.
```
Comment (aaronlehmann) [inline docs/swarm/manage-nodes.md]: "none" should be "node".

Comment (aaronlehmann) [inline docs/swarm/manage-nodes.md]: Actually, the sentence doesn't really make sense. `docker swarm leave` removes the node from the swarm, so it's not something you do before removing the node.

File: docs/swarm/manage-nodes.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+Node left the swarm.
+```
+
+When you remove a node from the swarm, the Docker Engine stops running in swarm
+mode. The orchestrator no longer schedules tasks to the node.
+
+If the node is a manager node, you will receive a warning about maintaining the
+quorum. To override the warning, pass the `--force` flag. If you leave
+the swarm as the last manager node, the swarm will become unavailable and you
```
Comment (aaronlehmann) [inline docs/swarm/manage-nodes.md]: "If the last manager node leaves the swarm..."

**Comment by aaronlehmann** [comment]
LGTM

**Comment by abronan** [comment]
LGTM

**Pull request merged**